src/components/account/ResetPassword.tsx

```diff
@@ -0,0 +1,67 @@
+import {SimpleTextInput, SimpleTextLabel} from '@components/widgets/StyledComponents'
+import {Alert, Button, Container, FormControl, Snackbar, Typography} from '@mui/material'
+import UserService from '@services/user.service'
+import {poppinsFont} from '@style/theme'
+import React from 'react'
+
+type ResetPasswordProps = {}
+export const ResetPassword: React.FunctionComponent<ResetPasswordProps> = () => {
+  const [email, setEmail] = React.useState('')
+  const [message, setMessage] = React.useState('')
+  const [showMessage, setShowMessage] = React.useState(false)
+
+  const emailInputId = 'email'
+
+  const handleRequestResetPassword = async (e: React.SyntheticEvent) => {
+    e.preventDefault()
+
+    await UserService.requestResetPassword(email)
+      .then(result => {
+        setMessage(result.data.message)
+      })
+      .catch((err: Error) => {
+        setMessage(err.message)
+      })
+      .finally(() => {
+        setShowMessage(true)
+      })
```

Comment:
Should not use both `await` and `.then/.catch/.finally` callbacks.

Either remove `await` or keep it and wrap in `try { } catch(e) {} finally {}`